app/models/physical_storage.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalStorage < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :physical_storages
```

Comment:
You need to add the physical_storages association to ext_management_system for this to work.

Reply:
Just fixed it to following what was done in [ManageIQ/manageiq#17423](https://github.com/ManageIQ/manageiq/pull/17423). Now Physical Infra Manager have many `physical_storages`.